src/box/txn.c

```diff
@@ -1233,6 +1229,12 @@ txn_commit_impl(struct txn *txn, enum txn_commit_wait_mode wait_mode)
 			txn_limbo_ack(&txn_limbo, txn_limbo.owner_id,
 				      limbo_entry->lsn);
```

Comment:
`wait_mode='none'` guarantees that there won't be any WAL writes. No yields. But `txn_limbo_ack()` can make a `CONFIRM` WAL write. We need to make is async for `wait_mode='none'`.

Reply:
Fixed: after #10628 we can return immediately after journal request submission.